app/models/podcast_import.rb

```diff
@@ -148,6 +149,23 @@ def create_podcast
     podcast
   end
 
+  def enclosure_prefix(podcast_item)
+    prefix = ''
+    link = [podcast_item.feedburner_orig_enclosure_link,
+            podcast_item.enclosure.try(:url),
+            podcast_item.media_contents.first.try(:url)].find do |url|
+              url.try(:match, /podtrac/) || url.try(:match, /blubrry/)
+            end
+    if link.try(:match, /podtrac/)
+      podtrac_end = link.index('mp3') + 'mp3/'.length
+      prefix += link[0...podtrac_end]
+    end
+    if link.try(:match, /blubrry/) && !prefix.match(/blubrry/)
+      prefix += 'media.blubrry.com/'
```

Comment:
https://create.blubrry.com/resources/podcast-media-download-statistics/instructions-for-developers/